src/test/java/core/basesyntax/CustomMathPowTest.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+public class CustomMathPowTest {
+
+    @Test
+    public void pow() {
+        for (int number = 0; number < 10; number++) {
+            for (int exponential = 10; exponential > 0; exponential--) {
+                CustomMathPow customMathPow = new CustomMathPow();
+                int actualResult = customMathPow.pow(number, exponential);
+                int expectedResult = (int) Math.pow(number, exponential);
+
+                Assert.assertEquals("Test failed with number = " + number
```

Comment:
line longer than 100 characters. In this case line breaks should be before each argument.